addons/web/static/src/views/fields/progress_bar/progress_bar_field.scss

```diff
@@ -4,6 +4,13 @@
         height: 15px;
         border: 1px solid $o-gray-300;
         background-color: $o-view-background-color;
+
+        > div {
+            background-color: $o-brand-primary;
+            &.o_progress_exceed {
+                background-color: #4A4F59;
```

Comment:
Maybe use the corresponding SCSS variable (which actually doesn't give exactly the same "gray" in Community vs Enterprise but :shrug: ) : `$o-gray-700` ?

Reply:
I think we should not hard code color here sir